src/rajAJun20/PrimeNumberOperations.java

```diff
@@ -0,0 +1,66 @@
+package rajAJun20;
+
+public class PrimeNumberOperations {
+	
+	void numberIsPrimeOrNot(int number){
+		boolean flag = false;
+		for (int index = 2; index < number/Math.sqrt(number); index++) {
+			if (number % 2 == 0) {
+				flag = true;
+				break;
+			}
+		}
+		if (flag == true) {
+			System.out.println("Number "+number+ " is not Prime");
+		}
+		else 
+			System.out.println("Number "+ number+ " is Prime");
+	}
+	
+	int totalCountOfPrimeNumbers(){
+		int count = 0;
+		for (int i = 2; i < 100; i++) {
+			boolean flag = false;
+			for (int j = 2; j < i ; j++) {
+				if(i % j == 0 ) {
```

Comment:
Here you should use numberIsPrimeOrNot() method.

Reply:
Changed method to print number is prime or not and called primeNumber method to show count and sum of prime numbers